lib/ansible/plugins/callback/debug2.py

```diff
@@ -0,0 +1,54 @@
+# (c) 2017 Ansible Project
```

Comment:
I don't think this should be in this PR. Please remove it, and create another PR for it if you want to add it to Ansible.